firefox-ios/Ecosia/Account/Auth/CredentialsManager/CredentialsManagerWrapper.swift

```diff
@@ -0,0 +1,42 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Auth0
+
+/// A wrapper class for `CredentialsManager` conforming to `CredentialsManaging`.
+///
+/// This class allows `CredentialsManager` to be injected as a dependency
+/// and enables testing by replacing it with a mock implementation.
+final class CredentialsManagerWrapper: CredentialsManaging {
```

Comment:
What is the advantage for having `CredentialsManagerWrapper`?

Since `CredentialsManager` directly conforms to `CredentialsManaging`, can't we simplify this to directly use it? The mock already directly conforms to that anyway too. And `Auth.defaultCredentialsManager` already stores that. Or am I missing something?

Reply:
Purely to make it testable. 
However, it was indeed inherent to a first attempt to make the Manager somehow storage-agnostic. I removed it now. Thanks for the suggestion 💪